docs/topics/braintree.rst

```diff
@@ -10,16 +10,47 @@ Calls braintree `ClientToken.generate <https://developers.braintreepayments.com/
 
 .. http:post:: /braintree/token/generate/
 
-    **Request**
+    :>json string token: the token returned by Braintree.
+    :status 200: token successfully generated.
+
+Create a customer
+-----------------
+
+Creates a customer in Braintree and the corresponding buyer in solitude. If the
+buyer exists in solitude already, it is not created. If the customer exists in
+Braintree already, it is not created.
 
-    No parameters.
+.. http:post:: /braintree/customer/
 
-    **Response**
+    :<json string uuid: the uuid of the buyer in solitude.
```

Comment:
Could you prefix this with ``**Request**`` so it's more clear? And below, I think ``**Response**`` instead of 'Returns...' would be more clear as well.